backend/src/integration-test/kotlin/com/tngtech/apicenter/backend/connector/rest/controller/SpecificationControllerIntegrationTest.kt

```diff
@@ -28,7 +28,7 @@ internal class SpecificationControllerIntegrationTest {
 
     @Test
     fun findAllSpecifications_shouldReturnAllSpecifications() {
-        mockMvc.perform(get("/specifications").with(user("user")))
+        mockMvc.perform(get("/api/1.0/specifications").with(user("user")))
```

Comment:
Minor version numbers in the API are really unusual. I would rather not use it. Instead something like "v1" so that you can easily see what the number is meant to signify.